common/web/lm-worker/src/test/test-runner/web-test-runner.config.mjs

```diff
@@ -0,0 +1,57 @@
+// @ts-check
+import { devices, playwrightLauncher } from '@web/test-runner-playwright';
+import { defaultReporter, summaryReporter } from '@web/test-runner';
+import { esbuildPlugin } from '@web/dev-server-esbuild';
+import named from '@keymanapp/common-test-resources/test-runner-rename-browser.mjs'
+import { fileURLToPath } from 'url';
+import { dirname, resolve } from 'path';
+import { LauncherWrapper, sessionStabilityReporter } from '@keymanapp/common-test-resources/test-runner-stability-reporter.mjs';
+
+const dir = dirname(fileURLToPath(import.meta.url));
+const KEYMAN_ROOT = resolve(dir, '../../../../../../');
```

Comment:
Doing this, along with setting `rootDir` to it explicitly, allows us to work around https://github.com/modernweb-dev/web/issues/2720 and https://github.com/modernweb-dev/web/issues/2721.  Those are based in use of a relative path, but this pattern allows us to use an absolute one instead.